fintrack/fintrack/settings.py

```diff
@@ -117,7 +119,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Asia/Karachi'
```

Comment:
Again why are we changing default also it's recommended to store everything in UTC and convert according to necessary